### ISSUE TITLE
Choices field not marked optional in DjangoPatchMutation

### ISSUE THREAD

**Issue by martasd** [tOgg1/graphene-django-cud#5]
I have the following model and mutation:

```
class Report(BaseModel):
    workspace = ForeignKey(Workspace, on_delete=CASCADE)
    name = CharField(max_length=255)

    MODULES = (("EVENT", "Event"), ("TASK", "Task"), ("AGENDA", "Agenda"), ("BEO", "Beo"))
    module = CharField(max_length=32, choices=MODULES)
    author = ForeignKey(UserProfile, on_delete=CASCADE)
    config = JSONField()
```

```
class UpdateReport(DjangoPatchMutation):
    class Meta:
        model = Report
        type_name = "UpdateReportInput"


```

`module` is NOT optional even though every field should be optional by default in `DjangoPatchMutation` according to the documentation:

![image](https://user-images.githubusercontent.com/591911/70134155-c7667c80-1687-11ea-84af-bc8dbdd70e6e.png)

**Comment by tOgg1** [comment]
Thanks for the report.

This is a bug I am aware of and is caused by a bad caching of the generated enum-types.

I will fast-track the fix of this.

**Comment by martasd** [comment]
Excellent!

**Comment by tOgg1** [comment]
This should be fixed in 0.2.1.
